### ISSUE TITLE
Problem with python2-gobject 3.16

### ISSUE THREAD

**Issue by catalin-hritcu** [linuxmint/mintmenu#130]
`mintmenu` broke after Arch updated the `python2-gobject` package from 3.14 to 3.16 yesterday. The message error is as follows:
```
$ mintmenu               
/usr/lib/python2.7/site-packages/gi/overrides/Gtk.py:50: RuntimeWarning: You have imported the Gtk 2.0 module.  Because Gtk 2.0 was not designed for use with introspection some of the interfaces and API will fail.  As such this is not supported by the pygobject development team and we encourage you to port your app to Gtk 3 or greater. PyGTK is the recomended python module to use with Gtk 2.0
  warnings.warn(warn_msg, RuntimeWarning)
Traceback (most recent call last):
  File "/usr/lib/linuxmint/mintMenu/mintMenu.py", line 6, in <module>
    from gi.repository import Gtk, GdkPixbuf, Gdk, GObject
  File "/usr/lib/python2.7/site-packages/gi/importer.py", line 159, in load_module
    dynamic_module = load_overrides(introspection_module)
  File "/usr/lib/python2.7/site-packages/gi/overrides/__init__.py", line 111, in load_overrides
    override_mod = importlib.import_module('gi.overrides.' + namespace)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/site-packages/gi/overrides/Gdk.py", line 161, in <module>
    class Event(Gdk.Event):
  File "/usr/lib/python2.7/site-packages/gi/overrides/Gdk.py", line 193, in Event
    Gdk.EventType.TOUCH_BEGIN: 'touch',
AttributeError: type object 'GdkEventType' has no attribute 'TOUCH_BEGIN'
```
Manually downgrading `python2-gobject` to 3.14 is a temporary workaround, but a better solution would be appreciated. Just looking at the error message, it's not clear at all whether this is `mintmenu`s fault though.

**Comment by catalin-hritcu** [comment]
I've also filed this against `pygobject`:
https://bugzilla.gnome.org/show_bug.cgi?id=747717

**Comment by catalin-hritcu** [comment]
This was just fixed in the just released pygobject 3.16.1. Closing.

**Comment by lmayorga1980** [comment]
I get the same error when but couldn't fine a solution within `homebrew` solution.
